native/cocos/audio/include/Common.h

```diff
@@ -0,0 +1,49 @@
+/****************************************************************************
```

Comment:
Rename to `Types.h`?

Reply:
I do think AudioDef.h is better, I will change to it anyway